### PR TITLE
add 2016-01-21 meeting notes

### PR DESCRIPTION
**Pull request by beaugunderson**

### CODE REVIEW

File: meetings/2016-01-21.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+### policy issue structure (#71)
+
+- ashley: bryan worked really hard on the policy issue structure, bryan do you
+  want to go over what you did there?
+- bryan: the idea is to have a new doc, POLICY_ISSUES.md, to provide structure
+  and explicit types of issues (questions, request for change, request for a
+  new policy)
+- bryan: also defines what type of information we expect for each issue, and
+  the focus (what we define as on- and off-topic)
+- bryan: charlotte also helped created magic links (links that create a new
```
Comment (scottgonzalez) [inline meetings/2016-01-21.md]: "helped created"

File: meetings/2016-01-21.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+### public/private spaces & cleaning up the slack (#81)
+
+- ashley: we have a form that's live that bryan & jona worked on
+- ashley: we've been talking about two roles, a member and a contributor role
+- ashley: it does seem clear that we like both the contributor and member roles
+  (which would require us to have both public/private spaces)
+- jeremiah: what are both roles?
+- ashley: members have a time commitment, collaborators want to help but do not
+  have a time commitment or access to private WG communications
+- (missed who said) one point is that collaborators are non-voting
```
Comment (Charlotteis) [inline meetings/2016-01-21.md]: I _think_ this was @nebrius [Bryan] ?

File: meetings/2016-01-21.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+- ashley: anything else?
+- ashley: there are several other communities getting codes of conduct in
+  place; if they wanted to reach out we could provide advice/support (like we
+  did with chakracore)
+- beau: i think that's great
+- bryan: we've learned a lot recently, i think we can help
+
+## open PRs
+
+- ashley: we don't have any! good job, go team! :sparkles:
```
Comment (Charlotteis) [inline meetings/2016-01-21.md]: great emoji skills :sparkles:

**Comment by ashleygwilliams** [comment]
grammar nit from @scottgonzalez, but otherwise LGTM, will merge once that's fixed

File: meetings/2016-01-21.md (hunk 2 above)
Comment (nebrius) [inline meetings/2016-01-21.md]: Yeah, that was me :). I know it's not a big deal for us in the group, but that kind of legalese is important to the foundation.

File: meetings/2016-01-21.md (hunk 3 above)
Comment (sup) [inline meetings/2016-01-21.md]: i concur

**Comment by sup** [comment]
LGTM after nits are dealt with!!!!

File: meetings/2016-01-21.md (hunk 2 above)
Comment (Charlotteis) [inline meetings/2016-01-21.md]: I have something to ask, but I'll go pop it over to the issue about this specifically.

**Comment by Charlotteis** [comment]
LGTM! Serious note taking skills!

**Comment by ashleygwilliams** [comment]
@beaugunderson is :muscle: :pencil2: :sparkles:

File: meetings/2016-01-21.md (hunk 2 above)
Comment (ashleygwilliams) [inline meetings/2016-01-21.md]: aye, it was

**Comment by beaugunderson** [comment]
@ashleygwilliams nits fixed!

**Comment by ashleygwilliams** [comment]
<3 mergin

**Pull request merged**